bindings/python/setup.py

```diff
@@ -90,6 +90,9 @@ def target_specific():
 	packages = ['libtorrent']
 
 else:
+	# Remove the '-Wstrict-prototypes' compiler option, which isn't valid for C++.
+	os.environ['OPT'] = ' '.join(
+		flag for flag in get_config_var('OPT').split() if flag != '-Wstrict-prototypes')
```

Comment:
it's a useful option for C files though. Do you know of a way to leave it in when building C?